js/util.js

```diff
@@ -111,4 +111,15 @@ export const reduceCount = (count, digits = 1) => {
 
   // Do not reduce if we are below 1000
   return divisor ? (count / divisor).toFixed(digits) + indicator : count
-}
\ No newline at end of file
+}
+
+export const SortingOptions = [
```

Comment:
would it make more sense for this to live in `constants.js` ?

Reply:
Moved into `constants.js`.